src/user-guide/installing-workflows.rst

```diff
@@ -0,0 +1,358 @@
++--------------------------------------+--------------------------------------+
+| cylc install --flow-name=new-name    |    ~/cylc-run/new-name/run1          |
++--------------------------------------+--------------------------------------+
+
+Any of the above commands may be run from anywhere on the file system with the
+addition of the option ``--directory=PATH/TO/SOURCE/DIRECTORY``, alternatively
+use ``-C PATH/TO/SOURCE/DIRECTORY``.
+
+Numbered Runs
+^^^^^^^^^^^^^
+
+By default, cylc install will install the workflow found in the current working
+directory into ``~/cylc-run/$(basename $PWD)/runN``, where runN = run1, run2, 
+run3,...
+
+``cylc install`` will automatically increment the run number of each install,
+provided the options ``--no-run-name`` or ``--run-name`` are not used. See
+:ref:`Command Line Options` for example behaviour.
+
+For convenience, a symlink to the highest (latest) numbered run will be created
+in the workflow directory, ``runN``.
+
+Example: A typical run directory structure, after three exectutions of 
+``cylc install`` will look as follows. 
+
+.. code-block:: bash
+
+   ├── _cylc-install
+   │   └── source -> /home/cylc-sources/test-flow
+   ├── run1
+   │   ├── flow.cylc
+   │   └── log
+   │       └── install
+   │           └── <time-stamp>-install.log
+   ├── run2
+   │   ├── flow.cylc
+   │   └── log
+   │       └── install
+   │           └── <time-stamp>-install.log
+   ├── run3
+   │   ├── flow.cylc
+   │   └── log
+   │       └── install
+   │           └── <time-stamp>-install.log
+   └── runN -> /home/cylc-run/test-flow/run3
+
+The numbered runs option may be overridden, using either the ``--no-run-name``
+or the ``--run-name`` options.
+
+
+Named Runs
+^^^^^^^^^^
+
+As an alternative to numbered runs, it is possible to name the runs, using the
+``--run-name`` option.
+In this case, the runN symlink will not be created.
+This option cannot be used in conjunction with numbered runs. 
+
+
+The Cylc Install Process
+------------------------
+
+There are two main parts of the ``cylc install`` process.
+
+1. File Installation
+
+2. Symlinking of Directories
+
+.. _File Installation:
+
+1. File Installation
+^^^^^^^^^^^^^^^^^^^^
+
+Installation will involved copying the files found in the source directory into
+a new run directory. If you wish to install files into an existing run
+directory, use ``cylc reinstall``, see :ref:`Reinstalling a Workflow`.
+
+
+.. _example_installation:
+
+Excluding Items From Installation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, cylc install will exclude `.git`, `.svn` directories.
+To configure excluded files and directories from the file installation,
+create a `.cylcignore` file in your source directory, this supports
+pattern matching.
+
+For example:
+We will look at running the cylc install command inside the directory
+`~/cylc-sources/test-flow` with the following directory structure:
+
+.. code-block:: bash
```

Comment:
Use `console` rather than bash for interactive session output.

```
# this is bash
ls $HOME

# this is console
$ ls $HOME
```

```suggestion
.. code-block:: console
```

Oddly, with Python you can use `python` for both Python code and interactive sessions.